### ISSUE TITLE
Brew shows "Error: SHA256 mismatch"

### ISSUE THREAD

**Issue by nitrocode** [cristianoliveira/ergo#13]
```
Error: SHA256 mismatch
Expected: b46e32dbc38523073728ee53a2db06dfed36e30c935a7dabb3e15b93dd8f6bcc
ergo
v0
1
5
darwin
amd64
tar
gz
Actual: b46e32dbc38523073728ee53a2db06dfed36e30c935a7dabb3e15b93dd8f6bcc
Archive: .../ergo-v0.1.5.tar.gz
```

**Comment by cristianoliveira** [comment]
Hey @nitrocode sorry for that. The last shasum I generated in a Linux system I think that may be the problem. I just released version `v0.1.6` please use `brew tap cristianoliveira/tap` and `brew install ergo` again.

**Comment by nitrocode** [comment]
It worked. Thank you
